plugins/httpapi/nd.py

```diff
@@ -261,7 +269,24 @@ def send_file_request(self, method, path, file=None, data=dict(), remote_path=No
                 self.nd.fail_json(msg="Cannot use requests_toolbelt MultipartEncoder() because requests_toolbelt module is not available")
```

Comment:
This error needs to be raised differently because `nd` is not part of the connection plugin. What type of error should we raise here?